services/wallet/inputs.go

```diff
@@ -427,3 +428,58 @@ func (blr *BitFlyerLinkingRequest) HandleErrors(err error) *handlers.AppError {
 	}
 	return handlers.ValidationError("bitflyer deposit wallet linking request validation errors", issues)
 }
+
+// Config defines a GeoLocationValidator configuration.
+type Config struct {
+	bucket string
+	object string
+}
+
+// GeoLocationValidator defines a GeoLocationValidator.
+type GeoLocationValidator struct {
+	s3     aws.S3
+	config Config
+}
+
+// NewGeoLocationValidator creates a new instance of NewGeoLocationValidator.
+func NewGeoLocationValidator(s3 aws.S3, config Config) *GeoLocationValidator {
+	return &GeoLocationValidator{
+		s3:     s3,
+		config: config,
+	}
+}
+
+// Validate checks to see if the given geolocation is enabled.
+func (g GeoLocationValidator) Validate(ctx context.Context, geolocation string) (bool, error) {
```

Comment:
Could use the same validate method that the handler parameter uses for ISO country codes.